pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/ModularLoadManager.java

```diff
@@ -140,4 +140,8 @@ default void writeBrokerDataOnZooKeeper(boolean force) {
      * @return bundle data
      */
     BundleData getBundleDataOrDefault(String bundle);
+
+    void setNamespaceBundleAffinity(String bundle, String broker);
+
+    String removeNamespaceBundleAffinity(String bundle);
```

Comment:
ideally, if we have `set` api then we can avoid `delete` API. do we really need delete API?  can we just call `set` with `null` value for removing affinity.

Reply:
removeNamespaceBundleAffinity also serves the purpose of get heree

Reply:
I mean instead of calling remove, can we just call `set(bundle, null)` -> null broker means remove affinity.?